<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitnessRoutine</class>
 <widget class="QWidget" name="FitnessRoutine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 lightgreen, stop:1 darkgreen);
}
</string>
  </property>
  <widget class="QLabel" name="FitnessRoutinLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Fitness Routine Tracker&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="MoodList">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Happy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tired</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Worried</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Productive</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="DescriptionforFitness">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>361</width>
     <height>74</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Hey, tell us your mood right now and the workout day that you would like to go with, then we'll suggest you a plan for it.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="WrorkoutRoutinelabel">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>220</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Workout Routine Day&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="UpperBodyButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Upper Body</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="LowerBodyButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lower Body</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="CardioButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cardio/HIIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="MoodStatusLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mood Status: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="SuggestPlanButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Suggest Plan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BacktoTabsfromFitness">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Tabs!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
